workshops/templates/workshops/training_progresses_inline.html

```diff
@@ -12,6 +12,6 @@
   </a>
   &nbsp;
 {% endfor %}
-<a href="{% url 'trainingprogress_add' person.id %}?next={{ request.get_full_path|urlencode }}"
+<a href="{% url 'trainingprogress_add' %}?trainee={{ person.id }}&next={{ request.get_full_path|urlencode }}"
```

Comment:
Maybe we could have a new `url` template tag that generates query string as well?